downstream/modules/platform/con-operator-custom-resources.adoc

```diff
@@ -3,3 +3,13 @@
 = Custom resources
 
 You can define custom resources for each primary installation workflows.
+
+//[Jameria] Moved this topic from supported installation section to custom resources since that's what the cross-referenced topic links to in the appendix (Custom resources appendix)
```

Comment:
@Alex-Izquierdo - I moved this topic from [1.3 Supported installation scenarios for Red Hat OpenShift Container Platform](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#ocp-supported-install_operator-install-planning) to [1.4 Custom resources](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#con-operator-custom-resources_operator-install-planning) since that's where the EDA_MAX_RUNNING_ACTIVATIONS fits into the appendix.